### PR TITLE
ENG-6816: Update IDP Okta module with new sso integration resource (CLOSED)

### PR DESCRIPTION
**Pull request by VictorGFM**
## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] Jira issue referenced in commit message and/or PR title

### CODE REVIEW

**Review by wcmjunior — COMMENTED** [review]
Please make sure to add to the docs that this Okta module will only work for CPs >= 2.24.

File: output.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+output "integration_saml_okta_id" {
+  description = "The ID (Alias) of the Okta SAML Integration resource."
+  value = cyral_integration_saml_okta.this.id
+}
+
+output "okta_app_saml_id" {
+  description = "The ID of the Okta SAML Application resource."
+  value = okta_app_saml.this.id
+}
```
Comment (wcmjunior) [inline output.tf]:
```suggestion
}

```

File: variables.tf [hunk 2]
```diff
@@ -1,17 +1,17 @@
-variable "tenant" {
+variable "control_plane" {
```
Comment (wcmjunior) [inline variables.tf]: Inverting positions for variables are just creating unnecessary changes. Is there a reason I did not see?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.terraform
+.terraform.lock.hcl
+terraform.tfstate*
```
Comment (wcmjunior) [inline .gitignore]:
```suggestion
terraform.tfstate*

```

File: variables.tf (hunk 2 above)
Comment (VictorGFM) [reply variables.tf]: Not really, I just thought that It would be more organized if ordered this way. I've reverted this change.

**Comment by VictorGFM** [comment]
This PR was closed and replaced by the new one #7.
